resource_packs/packs.json

```diff
@@ -1083,6 +1108,11 @@
 					"id": "xisumas_hermitcraft_base_season_8",
 					"name": "Xisuma's Hermitcraft Base (Season 8)",
 					"description": "Replaces the default Minecraft menu panorama with a 360° view of Xisuma's base from Hermitcraft Season 8."
+				},
+				{
+					"id": "xisumas_hermitcraft_base_season_9",
```

Comment:
Could you also rename the Xisuma's packs to be only "Season X", so they are the same as in VT right now?

Reply:
same for id's being only season_x